Extension/cpp.hint

```diff
@@ -1,2 +1,2 @@
-// https://go.microsoft.com/fwlink/?linkid=865984
-#pragma inject_default_hints
+// https://go.microsoft.com/fwlink/?linkid=865984

```

Comment:
What is the change here?

Reply:
It seems like it shouldn't be changed. The file is a copy of the "source" that is in the closed source repo so it shouldn't be different from that.

Reply:
Uh... this probably wasn't supposed to be here.

Reply:
grrr. one second...

Reply:
Fixed it